services/storage/docker-compose.yml

```diff
@@ -85,12 +85,15 @@ services:
       - ./04.key:/04.key
       - ./../../vendor/hosts:/etc/hosts
       - storage_s04:/storage
+      - ./s04tls.crt:/tls.crt
```

Comment:
If we assume those files exist, maybe we should have them in repo already and allow user to re-generate that before adding to trusted list?

Reply:
Then after starting the containers some git files will be modified. Is it ok?